### PR TITLE
Update operators.csv

### PR DESCRIPTION
**Pull request by mahtin**
Added globally understandable names to various ISPs

### CODE REVIEW

**Comment by lspgn** [comment]
Would prefer we keep one name (can drop France Telecom and DTAG)

**Comment by mahtin** [comment]
Now sorted. I would strongly recommend the use of the longer names (like France Telecom/Orange) as that provides the casual viewer a clear understanding of who the network is. While the Orange name change was a few years ago, with this revised listing you can clearly see the German, French, Italian networks called out by full names.

**Comment by jejenone** [comment]
France Telecom does not exist anymore as an entity. The legal name is Orange, even in France.

File: public/data/operators.csv [hunk 1]
```diff
@@ -9,16 +9,15 @@ GTT,transit,filtering peers only,partially safe,3257
 PCCW,transit,filtering peers only,partially safe,3491
 TATA,transit,filtering peers only,partially safe,6453
 Comcast,ISP,,unsafe,7922
-Deutsche Telekom,ISP,started,unsafe,3320
+Deutsche Telekom/DTAG,ISP,started,unsafe,3320
```
Comment (jejenone) [inline public/data/operators.csv:12]: I don't think DTAG adds any value here.

File: public/data/operators.csv [hunk 2]
```diff
@@ -9,16 +9,15 @@ GTT,transit,filtering peers only,partially safe,3257
...
+France Telecom/Orange,transit,started,unsafe,5511
```
Comment (jejenone) [inline public/data/operators.csv:13]: France Telecom as a legal entity and as a brand don't exist anymore.

File: public/data/operators.csv [hunk 3]
```diff
@@ -9,16 +9,15 @@ GTT,transit,filtering peers only,partially safe,3257
...
 Google,cloud,,unsafe,15169
 Hurricane Electric,transit,,unsafe,6939
 Level3/CenturyLink,transit,,unsafe,3356
-Orange,transit,started,unsafe,5511
-Sparkle,transit,started,unsafe,6762
 Sprint,transit,,unsafe,1239
+Telecom Italia/Sparkle,transit,started,unsafe,6762
```
Comment (jejenone) [inline public/data/operators.csv:18]: I'm ok with this one.

**Comment by mahtin** [comment]
Removed France Telecom name. 
I believe DTAG is an important tag for German users.

File: public/data/operators.csv (hunk 3 above)
Comment (adamschwartz) [reply public/data/operators.csv:18]: Seems more clear the other way

**Pull request closed without merging**